frontend/styles/components/admin/Card.module.scss

```
@@ -0,0 +1,40 @@
+@import '../../Variables.module.scss';
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)